symengine/parser.cpp

```diff
@@ -196,6 +208,31 @@ class ExpressionParser
                                  parse_string(iter + 1, operator_end[iter]));
                     iter = operator_end[iter] - 1;
 
+                } else if (s[iter] == '=') {
+                    result = Eq(result,
+                                parse_string(iter + 1, operator_end[iter]));
+                    iter = operator_end[iter] - 1;
+
+                } else if (s[iter] == '<' and s[iter + 1] == '=') {
```

Comment:
I had tried changing the `set<char> OPERATORS` to `set<std::string> OPERATORS` and subsequently `std::map<char, int> op_precedence` to `std::map<std::string, int> op_precedence`. But these changes would require an overhaul of the current iterative algorithm. I'd like to open an issue, for now, and tackle it later.